<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>555</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/res/IMG/1.png</normaloff>:/res/IMG/1.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(154, 45, 126), stop:1 rgb(247, 72, 118));</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_result">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>271</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 35px 0 0 0;
font-size: 47px;
color: #e0e0e0;
qproperty-alignment: AlignRight;
font-family: Arial Narrow;
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(222, 61, 79), stop:1 rgb(248, 83, 78));</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_process">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-top: 1px solid #adadad;
padding: 3px 5px;
 color: #8c8a8a;
font-size: 20px;
font-family: Arial Narrow;
qproperty-alignment: AlignRight; 
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(221, 59, 80), stop:1 rgb(225, 81, 90));
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_arrows">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(221, 59, 80), stop:1 rgb(225, 81, 90));
border-top: 1px solid #d2d2d2;
font-size: 20px;
color: #ffffff;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>131</y>
      <width>271</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_21">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
font-family: Arial Narrow;; 
  background-color: transparent;
   color: #e0e0e0;
  border-bottom: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(222, 61, 79), stop:1 rgb(248, 83, 78)); 
  border-right: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(222, 61, 79), stop:1 rgb(248, 83, 78)); 
  font-size: 27px;
  border-radius: 0;
  opacity: 0.9;
}

 QPushButton:pressed {
   background-color: rgb(175, 45, 62);
 }</string>
       </property>
       <property name="text">
        <string>←</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_24">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
font-family: Arial Narrow;; 
  background-color: transparent;
   color: #e0e0e0;
  border-bottom: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(222, 61, 79), stop:1 rgb(248, 83, 78)); 
  border-right: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(222, 61, 79), stop:1 rgb(248, 83, 78)); 
  font-size: 27px;
  border-radius: 0;
  opacity: 0.9;
}

 QPushButton:pressed {
   background-color: rgb(175, 45, 62);
 }</string>
       </property>
       <property name="text">
        <string>(</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_22">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
font-family: Arial Narrow;; 
  background-color: transparent;
   color: #e0e0e0;
  border-bottom: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(222, 61, 79), stop:1 rgb(248, 83, 78)); 
  border-right: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(222, 61, 79), stop:1 rgb(248, 83, 78)); 
  font-size: 27px;
  border-radius: 0;
  opacity: 0.9;
}

 QPushButton:pressed {
   background-color: rgb(175, 45, 62);
 }</string>
       </property>
       <property name="text">
        <string>)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="pushButton_23">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
font-family: Arial Narrow;; 
  background-color: transparent;
   color: #e0e0e0;
  border-bottom: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(222, 61, 79), stop:1 rgb(248, 83, 78)); 
  border-right: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(222, 61, 79), stop:1 rgb(248, 83, 78)); 
  font-size: 27px;
  border-radius: 0;
  opacity: 0.9;
}

 QPushButton:pressed {
   background-color: rgb(175, 45, 62);
 }</string>
       </property>
       <property name="text">
        <string>CE</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="pushButton_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
font-family: Arial Narrow;; 
  background-color: transparent;
   color: #e0e0e0;
  border-bottom: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(222, 61, 79), stop:1 rgb(248, 83, 78)); 
  border-right: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(222, 61, 79), stop:1 rgb(248, 83, 78)); 
  font-size: 27px;
  border-radius: 0;
  opacity: 0.9;
}

 QPushButton:pressed {
   background-color: rgb(175, 45, 62);
 }</string>
       </property>
       <property name="text">
        <string>÷</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
font-family: Arial Narrow;; 
  background-color: transparent;
   color: #e0e0e0;
  border-bottom: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(222, 61, 79), stop:1 rgb(248, 83, 78)); 
  border-right: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(222, 61, 79), stop:1 rgb(248, 83, 78)); 
  font-size: 27px;
  border-radius: 0;
  opacity: 0.9;
}

 QPushButton:pressed {
   background-color: rgb(175, 45, 62);
 }</string>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_8">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
font-family: Arial Narrow;; 
  background-color: transparent;
   color: #e0e0e0;
  border-bottom: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(222, 61, 79), stop:1 rgb(248, 83, 78)); 
  border-right: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(222, 61, 79), stop:1 rgb(248, 83, 78)); 
  font-size: 27px;
  border-radius: 0;
  opacity: 0.9;
}

 QPushButton:pressed {
   background-color: rgb(175, 45, 62);
 }</string>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButton_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
font-family: Arial Narrow;; 
  background-color: transparent;
   color: #e0e0e0;
  border-bottom: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(222, 61, 79), stop:1 rgb(248, 83, 78)); 
  border-right: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(222, 61, 79), stop:1 rgb(248, 83, 78)); 
  font-size: 27px;
  border-radius: 0;
  opacity: 0.9;
}

 QPushButton:pressed {
   background-color: rgb(175, 45, 62);
 }</string>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="pushButton_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
font-family: Arial Narrow;; 
  background-color: transparent;
   color: #e0e0e0;
  border-bottom: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(222, 61, 79), stop:1 rgb(248, 83, 78)); 
  border-right: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(222, 61, 79), stop:1 rgb(248, 83, 78)); 
  font-size: 27px;
  border-radius: 0;
  opacity: 0.9;
}

 QPushButton:pressed {
   background-color: rgb(175, 45, 62);
 }</string>
       </property>
       <property name="text">
        <string>√</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="pushButton_7">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
font-family: Arial Narrow;; 
  background-color: transparent;
   color: #e0e0e0;
  border-bottom: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(222, 61, 79), stop:1 rgb(248, 83, 78)); 
  border-right: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(222, 61, 79), stop:1 rgb(248, 83, 78)); 
  font-size: 27px;
  border-radius: 0;
  opacity: 0.9;
}

 QPushButton:pressed {
   background-color: rgb(175, 45, 62);
 }</string>
       </property>
       <property name="text">
        <string>×</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_9">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
font-family: Arial Narrow;; 
  background-color: transparent;
   color: #e0e0e0;
  border-bottom: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(222, 61, 79), stop:1 rgb(248, 83, 78)); 
  border-right: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(222, 61, 79), stop:1 rgb(248, 83, 78)); 
  font-size: 27px;
  border-radius: 0;
  opacity: 0.9;
}

 QPushButton:pressed {
   background-color: rgb(175, 45, 62);
 }</string>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_12">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
font-family: Arial Narrow;; 
  background-color: transparent;
   color: #e0e0e0;
  border-bottom: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(222, 61, 79), stop:1 rgb(248, 83, 78)); 
  border-right: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(222, 61, 79), stop:1 rgb(248, 83, 78)); 
  font-size: 27px;
  border-radius: 0;
  opacity: 0.9;
}

 QPushButton:pressed {
   background-color: rgb(175, 45, 62);
 }</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButton_10">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
font-family: Arial Narrow;; 
  background-color: transparent;
   color: #e0e0e0;
  border-bottom: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(222, 61, 79), stop:1 rgb(248, 83, 78)); 
  border-right: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(222, 61, 79), stop:1 rgb(248, 83, 78)); 
  font-size: 27px;
  border-radius: 0;
  opacity: 0.9;
}

 QPushButton:pressed {
   background-color: rgb(175, 45, 62);
 }</string>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="pushButton_1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
font-family: Arial Narrow;; 
  background-color: transparent;
   color: #e0e0e0;
  border-bottom: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(222, 61, 79), stop:1 rgb(248, 83, 78)); 
  border-right: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(222, 61, 79), stop:1 rgb(248, 83, 78)); 
  font-size: 27px;
  border-radius: 0;
  opacity: 0.9;
}

 QPushButton:pressed {
   background-color: rgb(175, 45, 62);
 }</string>
       </property>
       <property name="text">
        <string> x²</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="pushButton_11">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
font-family: Arial Narrow;; 
  background-color: transparent;
   color: #e0e0e0;
  border-bottom: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(222, 61, 79), stop:1 rgb(248, 83, 78)); 
  border-right: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(222, 61, 79), stop:1 rgb(248, 83, 78)); 
  font-size: 27px;
  border-radius: 0;
  opacity: 0.9;
}

 QPushButton:pressed {
   background-color: rgb(175, 45, 62);
 }</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_13">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
font-family: Arial Narrow;; 
  background-color: transparent;
   color: #e0e0e0;
  border-bottom: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(222, 61, 79), stop:1 rgb(248, 83, 78)); 
  border-right: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(222, 61, 79), stop:1 rgb(248, 83, 78)); 
  font-size: 27px;
  border-radius: 0;
  opacity: 0.9;
}

 QPushButton:pressed {
   background-color: rgb(175, 45, 62);
 }</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButton_16">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
font-family: Arial Narrow;; 
  background-color: transparent;
   color: #e0e0e0;
  border-bottom: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(222, 61, 79), stop:1 rgb(248, 83, 78)); 
  border-right: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(222, 61, 79), stop:1 rgb(248, 83, 78)); 
  font-size: 27px;
  border-radius: 0;
  opacity: 0.9;
}

 QPushButton:pressed {
   background-color: rgb(175, 45, 62);
 }</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="pushButton_14">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
font-family: Arial Narrow;; 
  background-color: transparent;
   color: #e0e0e0;
  border-bottom: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(222, 61, 79), stop:1 rgb(248, 83, 78)); 
  border-right: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(222, 61, 79), stop:1 rgb(248, 83, 78)); 
  font-size: 27px;
  border-radius: 0;
  opacity: 0.9;
}

 QPushButton:pressed {
   background-color: rgb(175, 45, 62);
 }</string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="pushButton_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
font-family: Arial Narrow;; 
  background-color: transparent;
   color: #e0e0e0;
  border-bottom: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(222, 61, 79), stop:1 rgb(248, 83, 78)); 
  border-right: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(222, 61, 79), stop:1 rgb(248, 83, 78)); 
  font-size: 27px;
  border-radius: 0;
  opacity: 0.9;
}

 QPushButton:pressed {
   background-color: rgb(175, 45, 62);
 }</string>
       </property>
       <property name="text">
        <string>±</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QPushButton" name="pushButton_15">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
font-family: Arial Narrow;; 
  background-color: transparent;
   color: #e0e0e0;
  border-bottom: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(222, 61, 79), stop:1 rgb(248, 83, 78)); 
  border-right: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(222, 61, 79), stop:1 rgb(248, 83, 78)); 
  font-size: 27px;
  border-radius: 0;
  opacity: 0.9;
}

 QPushButton:pressed {
   background-color: rgb(175, 45, 62);
 }</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="pushButton_18">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
font-family: Arial Narrow;; 
  background-color: transparent;
   color: #e0e0e0;
  border-bottom: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(222, 61, 79), stop:1 rgb(248, 83, 78)); 
  border-right: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(222, 61, 79), stop:1 rgb(248, 83, 78)); 
  font-size: 27px;
  border-radius: 0;
  opacity: 0.9;
}

 QPushButton:pressed {
   background-color: rgb(175, 45, 62);
 }</string>
       </property>
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="pushButton_20">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
font-family: Arial Narrow;; 
  background-color: transparent;
   color: #e0e0e0;
  border-bottom: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(222, 61, 79), stop:1 rgb(248, 83, 78)); 
  border-right: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(222, 61, 79), stop:1 rgb(248, 83, 78)); 
  font-size: 27px;
  border-radius: 0;
  opacity: 0.9;
}

 QPushButton:pressed {
   background-color: rgb(175, 45, 62);
 }</string>
       </property>
       <property name="text">
        <string>1/x</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QPushButton" name="pushButton_19">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
font-family: Arial Narrow;; 
  background-color: transparent;
   color: #e0e0e0;
  border-bottom: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(222, 61, 79), stop:1 rgb(248, 83, 78)); 
  border-right: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(222, 61, 79), stop:1 rgb(248, 83, 78)); 
  font-size: 27px;
  border-radius: 0;
  opacity: 0.9;
}

 QPushButton:pressed {
   background-color: rgb(175, 45, 62);
 }</string>
       </property>
       <property name="text">
        <string>=</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_17">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
font-family: Arial Narrow;; 
  background-color: transparent;
   color: #e0e0e0;
  border-bottom: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(222, 61, 79), stop:1 rgb(248, 83, 78)); 
  border-right: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(222, 61, 79), stop:1 rgb(248, 83, 78)); 
  font-size: 27px;
  border-radius: 0;
  opacity: 0.9;
}

 QPushButton:pressed {
   background-color: rgb(175, 45, 62);
 }</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
